Formula/libkeccak.rb

```diff
@@ -0,0 +1,36 @@
+class Libkeccak < Formula
+  desc "Keccak-family hashing library"
+  homepage "https://github.com/maandree/libkeccak"
+  url "https://github.com/maandree/libkeccak/archive/1.1.4.tar.gz"
+  sha256 "40e27983ea63faf47071402b212da48101b1dae37ce2702da947361ee8d4a313"
+
+  depends_on "binutils" => :build
+  depends_on "coreutils" => :build
+  depends_on "make" => :build
+
+  def install
+    system "make", "LIBEXT=dylib", "LIBFLAGS=-dynamiclib"
+    system "make", "install", "LIBEXT=dylib", "LIBFLAGS=-dynamiclib", "PREFIX=#{prefix}"
+    fix_libkeccak_symlinks
+  end
+
+  def fix_libkeccak_symlinks
+    mv "#{lib}/libkeccak.dylib.1.1", "#{lib}/libkeccak.1.1.dylib"
+    ln_s "#{lib}/libkeccak.1.1.dylib", "#{lib}/libkeccak.1.dylib"
+    rm "#{lib}/libkeccak.dylib"
+    ln_s "#{lib}/libkeccak.1.1.dylib", "#{lib}/libkeccak.dylib"
+  end
```

Comment:
Can you please port these fixes upstream into the software's build system so that the burden of maintaining it does not fall onto homebrew maintainers?

Having logic in the makefile to detect macOS and generate properly named dylibs should be fairly straight forward.

Reply:
Hmmmm, I see you're not @maandree, so maybe you can't port the comments upstream... I created a comment on an open issue to see if this can get addressed in a saner manner. (I.e., they could provide a make variable for the soname so it doesn't come out in the wrong order.)